src/encoded/types/file.py

```diff
@@ -556,9 +559,9 @@ def upload_key(self, request: Request) -> str:
         ]
     })
     def file_access_status(self, status: str = 'in review') -> Optional[str]:
-        if status in ['public', 'released']:
```

Comment:
Not sure how `file_access_status` is used currently (I think by the frontend only) but this might need to be reviewed.